seed/csharp-sdk/basic-auth-environment-variables/snippet.json

```diff
@@ -0,0 +1,29 @@
+{
+    "types": {},
+    "endpoints": [
+        {
+            "example_identifier": null,
+            "id": {
+                "path": "/basic-auth",
+                "method": "GET",
+                "identifier_override": "endpoint_basic-auth.getWithBasicAuth"
+            },
+            "snippet": {
+                "type": "typescript",
+                "client": "using SeedBasicAuthEnvironmentVariables;\n\nvar client = new SeedBasicAuthEnvironmentVariablesClient(\"USERNAME\", \"PASSWORD\");\nawait client.BasicAuth.GetWithBasicAuthAsync();\n"
```

Comment:
Similar to the other comment for environment variables, but this is for basic auth.

Reply:
fixing this as well